Cannot compile on Postgres18 Docker Oficial Image

When following the instructions in the readme on GitHub I get the following error.

```
"Platform is: linux"
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wimplicit-fallthrough=3 -Wcast-function-type -Wshadow=compatible-local -Wformat-security -Wmissing-variable-declarations -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -Wno-stringop-truncation -g -g -O2 -Werror=implicit-function-declaration -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection -fno-omit-frame-pointer -fPIC -fvisibility=hidden -I. -I./ -I/usr/include/postgresql/18/server -I/usr/include/postgresql/internal -Wdate-time -D_FORTIFY_SOURCE=2 -D_GNU_SOURCE  -I/usr/include/libxml2     -c -o misc.o misc.c
misc.c: In function ‘str_ltrim’:
misc.c:18:11: error: implicit declaration of function ‘isspace’ [-Wimplicit-function-declaration]
   18 |     while(isspace(*s)) s++;
      |           ^~~~~~~
misc.c:14:1: note: include ‘<ctype.h>’ or provide a declaration of ‘isspace’
   13 | #include "misc.h"
  +++ |+#include <ctype.h>
   14 | 
make: *** [<builtin>: misc.o] Error 1
```

--- a/misc.c
+++ b/misc.c
@@ -9,6 +9,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <ctype.h>
 
 #include "misc.h"
 

